control-plane/pkg/reconciler/source/source.go

```diff
@@ -208,7 +208,7 @@ func (r *Reconciler) reconcileKind(ctx context.Context, ks *sources.KafkaSource)
 		logger.Debug("Updated dispatcher pod annotation")
 	}
 
-	return statusConditionManager.Reconciled()
+	return nil
```

Comment:
why this change?

Reply:
I renamed the `Reconciled` method to `Addressable` since what it was doing is setting the address.

KafkaSource is not addressable, so we don't need to call that method.